content/dstu2/condition.md

```diff
@@ -19,18 +19,36 @@ Search for Conditions that meet supplied query parameters:
 
 ### Parameters
 
- Name             | Required? | Type          | Description
-------------------|-----------|---------------|------------------------------------------------------------------
- `patient`        | Y         | [`reference`] | The patient who has the condition. Example: `12345`
- `category`       | N         | [`token`]     | The [category] of the condition. Example: `diagnosis`, `problem`
- `clinicalstatus` | N         | [`token`]     | The [clinical status] of the condition. Example: `resolved`
+ Name             | Required?                                                            | Type          | Description
+------------------|----------------------------------------------------------------------|---------------|----------------------------------------------------------------------------------------------------------------
+ `_id`            | This or `patient`, if populated all other parameters are not allowed | [`token`]     | The logical resource id associated with the resource. Example: `p12345`
+ `patient`        | This or `_id`                                                        | [`reference`] | The patient who has the condition. Example: `12345`
+ `category`       | N                                                                    | [`token`]     | The [category] of the condition. Example: `diagnosis`, `problem`, `health-concern`
+ `clinicalstatus` | N                                                                    | [`token`]     | The [clinical status] of the condition. Example: `resolved`
 
-Note: Currently `diagnosis` and `problem` category codes are supported. Code `diagnosis` is from the [condition-category] system and code `problem` is from the [argonaut extension-codes] system.
+Note: Currently `diagnosis`, `problem` and `health-concern` category codes are supported. Code `diagnosis` is from the [condition-category] system and codes `problem` and `health-concern` are from the [argonaut] system.
```

Comment:
This just feels overly confusing - they can look at the value sets if they want by clicking the links (they should be able to, at least). Good to call out supported categories. No need to mention which systems they're all from. No need to call out Argonaut as something separate.